### PR TITLE
docs: fix typo in authentication.md

### PR DESCRIPTION
**Pull request by chaficnajjar**

### CODE REVIEW

File: docs/source/features/authentication.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ users: (parent, args, context) => {
 }
 ```
 
-This example is a field in our schema named `users` that returns a list of users’ names. The `if` check on the first line of the function looks at the `context` generated from our request, checks for a `user` object, and if one doesn’t exist, returns `null` for the whole field.
+This example is a field in our schema named `users` that returns a list of users’ names. The `if` check on the first line of the function looks at the `context` generated from our request, checks for a `user` object, and if one doesn’t exist, returns `[]` for the whole field.
```
Comment (abernix) [inline docs/source/features/authentication.md]: While there is a typo, the typo is in the code sample above — not here in the text.  (Keep reading until the next paragraph. 😄)

Can you undo this change, and change the code sample above to use `return null;` instead of `return [];`?

Comment (chaficnajjar) [reply docs/source/features/authentication.md]: @abernix That makes sense, I'll do it. I can't edit this PR anymore because I deleted my fork so I'll close this PR and create a new one with the requested changes. I guess I should've kept my fork until this PR was merged.
